### PR TITLE
Add commodities/ Integration to Time-series Charts

### PR DESCRIPTION
**Pull request by tbosch82**
Integrate the commodities endpoint into the time-series charts and give the user options for selecting multiple commodities and multiple contracts for comparison. Based on issue #35. Trading overview was also merged into wide chart (issue #37).

### Charts:

- Trading
Trading

- Expanded
Chart1

### CODE REVIEW

**Comment by tbosch82** [comment]
@ericdudley Take a look at this when you get a chance. I'm not sure if we want to keep the overview exactly how I have it, but it's a start.

File: demo/src/util/util.ts [hunk 1]
```diff
@@ -40,6 +40,44 @@ export function getContractLabel(expCode: string): string {
   return month + " " + year;
 }
 
+export function isPricePositive(a: number[], b: number[]): boolean {
```
Comment (ericdudley) [inline demo/src/util/util.ts]: These could use some documentation.

File: demo/src/util/util.ts [hunk 2]
```diff
@@ -40,6 +40,44 @@ export function getContractLabel(expCode: string): string {
...
+  return (
+    !!a.slice(-1)[0] &&
+    !!b.slice(-1)[0] &&
+    a.slice(-1)[0].toFixed(2) > b.slice(-1)[0].toFixed(2)
+  );
+}
+
+export function isPriceNegative(a: number[], b: number[]): boolean {
+  return (
+    !!a.slice(-1)[0] &&
```
Comment (ericdudley) [inline demo/src/util/util.ts]: What happens if `a` or `b` are empty.

Comment (tbosch82) [reply demo/src/util/util.ts]: @ericdudley It returns false if either, or both, are empty. I added documentation and put this in there.

**Pull request merged**